src/database/validator.js

```diff
@@ -22,7 +22,7 @@ const validateInitConfig = async (config) => {
     if (!config[dbConfigKeys.dbType]) {
         throw new Error(`Invalid config: Missing required field - '${dbConfigKeys.dbType}'`);
     }
-    if (!['MongoDB', 'MySQL'].includes(config[dbConfigKeys.dbType])) {
+    if (!['MongoDB', 'MySQL', 'PostgreSQL'].includes(config[dbConfigKeys.dbType])) {
```

Comment:
Update README under features to show that PostgreSQL is supported and examples to include PostgreSQL sample 
configuration.